manage-data/data-store/data-streams/set-up-data-stream.md

```diff
@@ -10,28 +10,36 @@ products:
 
 # Set up a data stream [set-up-a-data-stream]
 
+The process of setting up a data stream in {{stack}} and {{serverless-full}} is similar, making use of their respective APIs. However, because {{serverless-short}} provides a built-in [data stream lifecycle](/manage-data/lifecycle/data-stream.md) mechanism and retention settings, there's no need for configuring index lifecycle management ({{ilm-init}}) options as you would in an {{stack}} deployment.
```

Comment:
```suggestion
The process of setting up a data stream in {{stack}} and {{serverless-full}} is similar, making use of their respective APIs. However, because {{serverless-short}} provides a built-in [data stream lifecycle](/manage-data/lifecycle/data-stream.md) mechanism and retention settings, you don't need to configure index lifecycle management ({{ilm-init}}) options as you do in an {{stack}} deployment.
```
I might rephrase it slightly but feel free to ignore.

---

Also, I experimented with putting the part about ILM into a note below Step 1 just to see how that would look. But, I think I like your way better. :-) 

<img width="584" height="483" alt="Image" src="https://github.com/user-attachments/assets/b5bc370a-5dd2-41a8-b0de-41168dd520b2" />